challenges/linkedList2/linkedListTes.java

```diff
@@ -0,0 +1,45 @@
+package linkedList;
+
+import org.junit.Test;
+
+import org.junit.Assert.*;
+
+
+
+public class LinkedListTest {
+
+    @Test
+    public void listEmptyTest(){
+        linkedList l = new LinkedList<Integer>();
+        assertEquals("new linked list is empty", l.isEmpty());
+    }
+
+    @Test public void listTestInsert(){
```

Comment:
Great work with the tests. Don't forget if you have multiple asserts with different method calls, they probably belong in a separate test! (talking about .insert here)